ipn/ipnlocal/local.go

```diff
@@ -4614,6 +4616,12 @@ func unmapIPPrefixes(ippsList ...[]netip.Prefix) (ret []netip.Prefix) {
 	return ret
 }
 
+var metricAdvertisedRoutes = usermetrics.NewMultiLabelMap[struct{}](
+	"tailscaled_advertised_routes",
+	"gauge",
+	"Number of subnet routes advertised by the node. (excluding exit node /0 routes)",
```

Comment:
For most users I suspect exit nodes and subnet routers are separate features, so I don't think we should mention exit nodes here.

```suggestion
	"Number of subnet routes advertised by the node",
```